clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -80,7 +80,7 @@
 
   private volatile Connection clusterConnection;
   private EhcacheClientEntityFactory entityFactory;
-  private EhcacheClientEntity entity;
+  EhcacheClientEntity entity;
```

Comment:
Did that for testability. Not sure if it's a good practice to expose the visibility of a member field just for testing purposes. Is accessing this field using reflection preferred over this approach?